src/cli/main.cpp

```diff
@@ -502,6 +467,9 @@ main(int argc, char** argv)
             ctx.mirror_map = parse_mirrors(ctx, config["mirrors"]);
         }
     }
+
+    powerloader::erase_duplicates(du_files);
```

Comment:
Is this new?

Reply:
Yes it's in this PR, I couldnt find an already-available equivalent. This call only filters input from cli.

Reply:
I was not sure why we need it now? :)